stdlib/public/core/Sequence.swift

```diff
@@ -1140,6 +1140,13 @@ extension Sequence {
   @inlinable
   public __consuming func _copyContents(
     initializing buffer: UnsafeMutableBufferPointer<Element>
+  ) -> (Iterator,UnsafeMutableBufferPointer<Element>.Index) {
+    return _copySequenceContents(initializing: buffer)
+  }
+
+  @inlinable
```

Comment:
This will need to be marked `@_alwaysEmitIntoClient` to prevent back deployment issues.

Reply:
What's better: force-inlining that or leaving that alone and duplicating the code in `Slice._copyContents`? It doesn't seem obvious to me.

Reply:
At the end of the day, I don't think it matters either way; but all things being equal, DRY is nice, so I weakly prefer having a single definition for both sites. (We do that in several places elsewhere.)